### PR TITLE
[C++ / en]  Added Tuple .

### PR DESCRIPTION
**Pull request by Jaskamalkainth**

### CODE REVIEW

File: c++.html.markdown [hunk 1]
```diff
@@ -955,6 +955,54 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+
+#include<tuple>
+
+// Conceptually, Tuples are similar to  old data structures (C-like structs) but instead of having named data members , 
+// its elements are accessed by their order in the tuple.
+
+// We start with constructing a tuple.
+//
+// Packing values into tuple
+auto first  = make_tuple ( 10 , 'A' ) ;
```
Comment (vendethiel) [inline c++.html.markdown]: Can you please fix the codestyle in this Pr? Thanks

Comment (Jaskamalkainth) [inline c++.html.markdown]: I'm not sure I understood your request correctly. Can you elaborate ?

Comment (vendethiel) [inline c++.html.markdown]: remove spaces, etc. This particular line should be `auto first = make_tuple(10, 'A');` (I removed the double space before the assignment, and the spaces in the parentheses)

Comment (Jaskamalkainth) [inline c++.html.markdown]: I corrected all the changes you mentioned and sent another PR. 
Let me know if anything else is to be corrected.
Thanks.
